src/button/internal.tsx

```diff
@@ -80,6 +81,18 @@ export const InternalButton = React.forwardRef(
     const { stepNumber, stepNameSelector } = useFunnelStep();
     const { subStepSelector, subStepNameSelector } = useFunnelSubStep();
 
+    usePerformanceMarks(
+      'primaryButton',
```

Comment:
Why "primaryButton" as opposed to "Button"?

Reply:
The alternative I see is using `button` and including the variant value in the mark details, but that might create the expectation that the mark is also emitted for non-primary buttons. If/when we expand the performance marks to more components/variants, I'd replace it with `button`.